pkgs/development/libraries/openssl/default.nix

```diff
@@ -122,6 +123,19 @@ let
       "MANSUFFIX=ssl"
     ];
 
+    buildFlags = lib.optionals static [
+      # Even though engines are disabled in static builds, we have to
+      # override ENGINESDIR so the bin output doesn't end up with an
+      # reference to the lib output.
+      "ENGINESDIR=/"
+    ];
+
+    installFlags = lib.optionals static [
```

Comment:
It would — it leaving a reference is why I need to change it in the first place.

Reply:
Another  thing that would be possible would be to `rm` the normal engines directory and use `remove-references`, but whatever you prefer works.

Reply:
I'm more comfortable with this, because it's limited in scope.  remove-references is a big hammer, and it would be very difficult to accidentally affect things I hadn't accounted for with it.